content/using_ec2_spot_instances_with_eks/spotworkers/selecting_instance_types.md

```diff
@@ -26,26 +26,85 @@ We can diversify Spot instance pools using two strategies:
-
-{{% notice note %}}
- The frequency of Spot Instance interruptions reflected in *Spot Instance Advisor* may change over time. Savings compared to On-Demand are calculated over the last 30 days. The above just provides a real world example from a specific time and will probably be different when you are performing this workshop. Note also that not all the instances are available in all the regions.
-{{% /notice %}}
-
-![Selecting Instance Type with 4vCPU and 16GB](/images/using_ec2_spot_instances_with_eks/spotworkers/4cpu_16_ram_instances.png)
-
-In this case with Spot Instance Advisor we can create a 4vCPUs_16GB nodegroup with the following diversified instances: **m5.xlarge, m5d.xlarge, m4.xlarge, m5a.xlarge, t2.xlarge, t3.xlarge, t3a.xlarge**
+Our goal in this workshop, is to create at least 2 diversified groups of instances that adhere the 1vCPU:4GB RAM ratio. 
+
+We will use **[amazon-ec2-instance-selector](https://github.com/aws/amazon-ec2-instance-selector)** to help us select the relevant instance
+types and familes with sufficient number of vCPUs and RAM. 
+
+There are over 270 different instance types available on EC2 which can make the process of selecting appropriate instance types difficult. **[amazon-ec2-instance-selector](https://github.com/aws/amazon-ec2-instance-selector)** helps you select compatible instance types for your application to run on. The command line interface can be passed resource criteria like vcpus, memory, network performance, and much more and then return the available, matching instance types.
+
+Let's first install **amazon-ec2-instance-selector** :
+
+```
+curl -Lo ec2-instance-selector https://github.com/aws/amazon-ec2-instance-selector/releases/download/v1.3.0/ec2-instance-selector-`uname | tr '[:upper:]' '[:lower:]'`-amd64 && chmod +x ec2-instance-selector
+sudo mv ec2-instance-selector /usr/local/bin/
+ec2-instance-selector --version
+```
+
+Now that you have ec2-instance-selector installed, you can run
+`ec2-instance-selector --help` to understand how you could use it for selecting
+instances that match your workload requirements. For the purpose of this workshop
+we need to first get a group of instances that meet the 4vCPUs and 16GB of RAM.
+Run the following command to get the list of instances.
+
+```bash
+ec2-instance-selector --vcpus 4 --memory 16384 --gpus 0 --current-generation -a x86_64 --deny-list '.*n.*'      
+```
+
+This should display a list like the one that follows (note results might differ depending on the region). We will use this instances as part of one of our node groups.
+
+```
+m4.xlarge
+m5.xlarge
+m5a.xlarge
+m5d.xlarge
+t2.xlarge
+t3.xlarge
+t3a.xlarge           
```

Comment:
Can we add a note re: usage of burstable instances and potential surplus credits charges?